src/coreComponents/physicsSolvers/fluidFlow/IsothermalCompositionalMultiphaseBaseKernels.hpp

```diff
@@ -1446,6 +1446,8 @@ struct StatisticsKernel
           arrayView3d< real64 const, multifluid::USD_PHASE > const & phaseDensity,
           arrayView4d< real64 const, multifluid::USD_PHASE_COMP > const & phaseCompFraction,
           arrayView2d< real64 const, compflow::USD_PHASE > const & phaseVolFrac,
+          arrayView2d< real64 const, compflow::USD_PHASE > const & phaseMassTrapped,
```

Comment:
```suggestion
          arrayView2d< real64 const, compflow::USD_PHASE > const & phaseTrappedVolFrac,
```
because I think this is the saturation, not yet the mass?